README.md

```diff
@@ -2,17 +2,101 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+1. Install dependencies:
+  ```bash
+  yarn
```

Comment:
with the yarn/npm stuff i think we just put yarn and if they wanna use npm they can? make it succinct